app/Repositories/Contracts/TopicRepository.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: quantien
+ * Date: 11/2/18
+ * Time: 12:58 PM
+ */
+
+namespace App\Repositories\Contracts;
+
+use App\Models\Topic;
+
+interface TopicRepository extends AbstractRepository
+{
+    public function getAll($data = []);
+
+    public function store($data = []);
+
+    public function find($id);
+
+    public function update($id, $data = []);
+
+    public function show($id);
+
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.EndFileNewline.NoneFound</br>**Severity**: error</br>**File**: app/Repositories/Contracts/TopicRepository.php</br>**Line**: 26</br>Expected 1 newline at end of file; 0 found